store/postgres/src/subgraph_store.rs

```diff
@@ -1111,6 +1111,22 @@ impl WritableStore {
             backoff.sleep_async().await;
         }
     }
+
+    /// Try to send a `StoreEvent`; if sending fails, log the error but
+    /// return `Ok(())`
+    fn try_send_store_event(&self, event: StoreEvent) -> Result<(), StoreError> {
+        if *SEND_SUBSCRIPTION_NOTIFICATIONS {
+            self.store
+                .send_store_event(&event)
+                .map_err(
+                    |e| error!(self.logger, "Could not send store event"; "error" => e.to_string()),
+                )
+                .ok();
```

Comment:
Why there is this `.ok` here?

The way I usually ignore `Result`s using `let _ = ...;`, but I'm not sure if that's what you're trying to do here, enlighten me 🙂

Reply:
That's a much better way (and all I was going for here) Changed.